data/mods/gen9ssb/moves.ts

```diff
@@ -145,6 +145,55 @@ export const Moves: {[k: string]: ModdedMoveData} = {
 		type: "Grass",
 	},
 
+	// Appletun a la Mode
+	extracourse: {
+		accuracy: true,
+		basePower: 0,
+		category: "Status",
+		shortDesc: "Recycles held berry and boosts 2 random stats (-acc/eva) by 1.",
+		name: "Extra Course",
+		gen: 9,
+		pp: 5,
+		priority: 0,
+		onTryMove() {
+			this.attrLastMove('[still]');
+		},
+		onPrepareHit(target, source) {
+			this.add('-anim', source, 'Recycle', target);
+		},
+		flags: {snatch: 1},
+		onHit(pokemon) {
+			if (pokemon.item || !pokemon.lastItem) return false;
+			const item = pokemon.lastItem;
+			pokemon.lastItem = '';
+			this.add('-item', pokemon, this.dex.items.get(item), '[from] move: Extra Course');
+			pokemon.setItem(item);
+			let stats: BoostID[] = [];
+			const boost: SparseBoostsTable = {};
+			let statPlus: BoostID;
+			for (statPlus in pokemon.boosts) {
+				if (statPlus === 'accuracy' || statPlus === 'evasion') continue;
+				if (pokemon.boosts[statPlus] < 6) {
+					stats.push(statPlus);
+				}
+			}
```

Comment:
dont immediately return here, just make the recycle happen if the pokemon doesnt have an item

Reply:
noted, fixing in next commit